### PR TITLE
updated star function to display accurate rating

### PR DESCRIPTION
**Pull request by ellenkramp**
I pulled the most recent master file after @ponchieponcho committed just now so this shouldn't override old changes anymore

@rfpoulos and @ponchieponcho  plz 👀

### CODE REVIEW

File: javascript/detail_and_listview.js [hunk 1]
```diff
@@ -157,10 +174,13 @@ var createRouteCardSkeleton = function(object, walkerLoc) {
     divReviewStars.setAttribute("class", "review-stars");
     divReviews.appendChild(divReviewStars);
 
-    getAmountOfStars(divReviewStars);
+    getAmountOfStars(divReviewStars, id);
```
Comment (rfpoulos) [inline javascript/detail_and_listview.js]: I don't understand what ID is and how it is different then object?

Comment (ellenkramp) [reply javascript/detail_and_listview.js]: I don't know what object was intended to be but when I tried to use it, it was giving me [object object]

Comment (ellenkramp) [reply javascript/detail_and_listview.js]: The id itself just represents the actual id string value

**Comment by rfpoulos** [comment]
I'm pretty sure object is the whole walk data, so you could grab the id of each card within the function to get a path to the rating.

**Comment by ellenkramp** [comment]
@rfpoulos what code would I use to refer to the value of the object's id? I tried object.val() and object.key() and neither of those worked

**Comment by ponchieponcho** [comment]
Seems like it gets the job done without breaking anything. I'm okay with merging 👍

**Pull request merged**